mlflow/store/model_registry/__init__.py

```diff
@@ -1,2 +1,5 @@
+# Path to default location for backend when using local FileStore.
+DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH = "./mlruns/models"
```

Comment:
Instead of setting the default path to `./mlruns/models` here, I think we should update the logic of the `FileStore` class to always write to / read from a `models` subdirectory of the path that the user specifies for their file store location. For example, if the user calls `mlflow.set_tracking_uri("/tmp/dir")`, then the current approach will still write experiments and models to the same directory.